Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.cpp

```diff
@@ -194,17 +194,32 @@ bool NumericCalculationNode::contains_percentage() const
     return m_value.has<Percentage>();
 }
 
+static Optional<CSSNumericType> numeric_type_from_calculated_style_value(CalculatedStyleValue::CalculationResult::Value const& value)
+{
+    return value.visit(
+        [](Number const&) -> Optional<CSSNumericType> { return {}; },
```

Comment:
I don't know what `CSSNumericType` should occur in the case of `Number`, I assume there isn't a set one and it can change. But this code should probably be corrected if there is one.

Reply:
This is actually a duplicate of what's done in `NumericCalculationNode::create()`, maybe that code can be pulled out and reused.

Reply:
Thanks for the suggestion :+1: